### PR TITLE
[DS3231]: add Class Member function setEpoch()

### PR DESCRIPTION
**Pull request by hasenradball**
# Add Menber function to set the RCT by passing the Epoch as parameter
As discussed in #37 addin new member Function to set the RTC via parameter.

## Remove one small compiler warning
Change a constant variable

### CODE REVIEW

File: DS3231.cpp [hunk 1]
```diff
@@ -127,7 +127,7 @@ DateTime::DateTime (uint16_t year, uint8_t month, uint8_t day, uint8_t hour, uin
 // supported formats are date "Mmm dd yyyy" and time "hh:mm:ss" (same as __DATE__ and __TIME__)
 DateTime::DateTime(const char* date, const char* time) {
    static const char month_names[] = "JanFebMarAprMayJunJulAugSepOctNovDec";
-   static const char buff[4]={'0','0','0','0'};
+   static char buff[4] = {'0','0','0','0'};
```
Comment (hasenradball) [inline DS3231.cpp:130]: remove a compiler warning.

**Comment by hasenradball** [comment]
Works great! 👍

**Comment by awickert** [comment]
Thank you for proposing and testing! I do not have hardware here to test it with, so I will accept the pull request. Please let me know if you think that this is ready based on your testing to generate a new release; if so, I will do this, and it will be taken up by Arduino for automatic installation via the IDE.

**Comment by awickert** [comment]
Thank you for proposing and testing! I do not have hardware here to test it with, so I will accept the pull request. Please let me know if you think that this is ready based on your testing to generate a new release; if so, I will do this, and it will be taken up by Arduino for automatic installation via the IDE.

**Comment by hasenradball** [comment]
I regonized some other compiler warnings during implementation which came up, but I was not able to fix it immediately.

**Comment by awickert** [comment]
Thanks for the note. I'll hold off on a new release until the compiler warnings are handled.

**Comment by hasenradball** [comment]
You can accept a new PR where I removed the compiler warnings.
see (#39).

**Pull request merged**